sockeye/training.py

```diff
@@ -14,12 +14,17 @@
 """
 Code for training
 """
+from collections import namedtuple
```

Comment:
Not anymore. _TrainingState was initially a namedtuple, but I updated to a class in order to be able to update the values.